components/centraldashboard/public/components/dashboard-view.pug

```diff
@@ -21,6 +21,10 @@ div#grid
     .column
         paper-card(heading='Recent Notebooks')
                 notebooks-card(namespace='[[namespace]]')
+        paper-card(heading='Recently Created Pipelines')
+            pipelines-card(artifact-type='pipelines')
+        paper-card(heading='Recent Pipeline Runs')
+            pipelines-card(artifact-type='runs')
```

Comment:
You know I never thought of that. Going with the inheritance route doesn't seem like it would provide much benefit since the main purpose of the `paper-card` is to get the elevation style and template which exposes the slot where our content goes. I experimented with it and it seemed like a bit more trouble than it's worth, especially since our overridden styles for paper-card wouldn't apply to the new element's name.

We could have the `paper-card` be part of the template for the `pipelines-card` and `notebooks-card` items, but we'd need to move some styles around to accommodate that.

It would probably be more explicit to say `pipelines-card-body` but I got lazy with naming.

Reply:
I see, yeah both are valid. Ok let's go with the body name in that case?